cfgov/v1/documents.py

```diff
@@ -130,7 +129,10 @@ def search_title(self, search):
 
     def order_results(self, search):
         total_results = search.count()
-        return search.sort('-date_published')[0:total_results]
+        if not self.title:
+            return search.sort('-date_published')[0:total_results]
+        else:
+            return search.sort(self.order_by)[0:total_results]
```

Comment:
Why does this depend on `title`?

Reply:
Honestly - I made the call because I felt ordering by relevance on a keyword filter wouldn't make much sense and only really has an impact when doing an actual match. I felt content editors would probably prefer the behavior this way but we should confirm the assumption.